pennylane/beta/devices/default_tensor.py

```diff
@@ -155,6 +155,13 @@ class DefaultTensor(Device):
     _zero_state = np.array([1.0, 0.0], dtype=C_DTYPE)
 
     def __init__(self, wires, shots=None, representation="exact", contraction_method="auto"):
+
+        warnings.warn(
+            "The default.tensor device is deprecated and due to be removed in an upcoming PennyLane release.",
```

Comment:
Would this make sense? This way it should warn for `default.tensor` and `default.tensor.tf` using their dedicated names.
```suggestion
            f"The {self.short_name} device is deprecated and due to be removed in an upcoming PennyLane release.",
```